pkg/ansible/run.go

```diff
@@ -162,12 +162,25 @@ func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	}
 	// Create Service object to expose the metrics port(s).
 	// TODO: probably should expose the port as an environment variable
-	_, err = metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
+	service, err := metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
 	if err != nil {
 		log.Error(err, "Exposing metrics port failed.")
 		return err
 	}
 
+	// CreateServiceMonitors will automatically create the prometheus-operator ServiceMonitor resources
+	// necessary to configure Prometheus to scrape metrics from this operator.
+	services := []*v1.Service{service}
+	_, err = metrics.CreateServiceMonitors(cfg, namespace, services)
+	if err == metrics.ErrServiceMonitorNotPresent {
```

Comment:
See that the golang scaffolded was changed indeed to not show log errors regards it when should not. See https://github.com/operator-framework/operator-sdk/pull/2190. So, could you please update the implementation in order to reflect the current one made in [go](https://github.com/operator-framework/operator-sdk/pull/2190/files#diff-6438d7f67aed0d3fd6727a1f27133faeR167)?

Reply:
I went ahead and made this ServiceMonitor creation block equivalent to what is found in [`internal/scaffold/cmd.go`](https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/cmd.go#L189-L200)

Reply:
I mean: Should we not? 
1. Add in the main func.

> 	// Add the Metrics Service
> 	addMetrics(ctx, cfg, namespace)

2. At the end add all addMetrics implementation

3. Add the customization requested by @fabianvf in the addMetrics

WDYT?

Reply:
Now I see what you are saying. I missed the `addMetrics` piece from the scaffold.

Reply:
I'm not sure about which customization request you are referring to from @fabianvf :
1. If `reason` vs `error`, this isn't needed anymore because we are testing with the servicemonitor CRD installed in the cluster
1. If referring to removing one of the `log.Info`'s .. I just don't agree that it's a valuable deviation from the scaffold. It would be one thing to me if this were in the Reconcile loop and it was an additional line of log output on every reconcile. However, in the majority of cases it will be two lines of output at the very beginning of operator startup saying "couldn't create serviceMonitor" and "install prometheus-operator to create ServiceMonitor objects".